packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -145,17 +211,15 @@ class BlockListBlock extends Component {
 					accessible={ ! isSelected }
 					accessibilityRole={ 'button' }
 				>
-					<View style={ [ styles.blockHolder, borderStyle, { borderColor } ] }>
-						{ showTitle && this.renderBlockTitle() }
-						<View
-							accessibilityLabel={ accessibilityLabel }
-							style={ [ ! isSelected && styles.blockContainer, isSelected && styles.blockContainerFocused ] }
-						>
-							{ isValid && this.getBlockForType() }
-							{ ! isValid &&
+					<View
+						pointerEvents={ isTouchable ? 'auto' : 'box-only' }
+						accessibilityLabel={ accessibilityLabel }
+						style={ this.applyBlockStyle() }
```

Comment:
This (and related functions that compute styles) could probably be moved to its own HoC for better clarity. I guess it's fine for a first iteration 👍

Reply:
Ok I will keep that in mind to refactor that in next approach